src/main/java/org/stellar/sdk/Transaction.java

```diff
@@ -376,6 +395,14 @@ public Builder setOperationFee(int operationFee) {
       return this;
     }
 
+    public Builder setNetwork(Network network) {
+      if (mNetwork != null) {
+        throw new RuntimeException("Network has already been configured.");
+      }
+      mNetwork = checkNotNull(network, "Network cannot be null");
+      return this;
+    }
```

Comment:
Do you think it's also worth adding this as a constructor param? This must always be set, as `sourceAccount`.